capstone.css

```diff
@@ -445,89 +445,89 @@ footer {
 
 @media only screen and (max-width: 768px) {
   .Hamburger {
-  display: inline-block;
-  height: 20px;
-  width: 20px;
+    display: inline-block;
+    height: 20px;
+    width: 20px;
   }
 
   .firsthead {
-  padding-bottom: 0;
+    padding-bottom: 0;
   }
 
   .firsthead h1 {
-  font-size: 30px;
+    font-size: 30px;
   }
 
   .program {
-  height: 930px;
+    height: 930px;
   }
 
   .logo {
-  display: none;
+    display: none;
   }
 
   .SocialBar {
- display: none;
+    display: none;
   }
 
   .navSmall {
- display: flex;
+    display: flex;
   }
 
   .All-Programs {
- grid-auto-flow: row;
- height: 900px;
- overflow-y: hidden;
+    grid-auto-flow: row;
+    height: 900px;
+    overflow-y: hidden;
```

Comment:
- [ ] Please try to improve the featured speakers' section design based on the design given. Take a look at the comparison below 👇 
![image](https://user-images.githubusercontent.com/90222110/191802170-97285071-fa15-49fc-b25c-5e507efb6e6a.png)


![image](https://user-images.githubusercontent.com/90222110/191802251-4036b404-bdc8-45ec-a282-9cbf14f94cd2.png)

Reply:
Ok. I will follow all of your reviews to correct my capstone project.

Again a lot of thanks to you.

Reply:
I made the font-size of the names bigger.